crypto/src/main/java/net/corda/v5/crypto/merkle/MerkleProof.java

```diff
@@ -18,11 +21,21 @@
  *
  */
 @CordaSerializable
-interface MerkleProof {
-    val proofType: MerkleProofType
-    val treeSize: Int
-    val leaves: List<IndexedMerkleLeaf>
-    val hashes: List<SecureHash>
+public interface MerkleProof {
+
+    /**
+     * Get the type of the proof
+     * 
+     * @return Type of the proof
+     */
+    @NotNull
+    MerkleProofType getProofType();
+
+    int getTreeSize();
+    
+     List<IndexedMerkleLeaf> getLeaves();
+    
+     List<SecureHash> getHashes();
```

Comment:
Looks like a couple of stray spaces? (nitpicky)

Reply:
fixed